opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -375,5 +380,305 @@ namespace Opm
+                            "WAT_DEN",
+                            Opm::UnitSystem::measure::density,
+                            adbToDoubleVector(sd.rq[aqua_idx].rho));
+                }
+                if (liquid_active) {
+                    simProps.emplace_back(
+                            "OIL_DEN",
+                            Opm::UnitSystem::measure::density,
+                            adbToDoubleVector(sd.rq[liquid_idx].rho));
+                }
+                if (vapour_active) {
+                    simProps.emplace_back(
+                            "GAS_DEN",
+                            Opm::UnitSystem::measure::density,
+                            adbToDoubleVector(sd.rq[vapour_idx].rho));
+                }
+            }
+
+            /**
+             * Viscosities for water, oil gas
+             */
+            if (outKeywords["VISC"] > 0) {
+                if (aqua_active) {
+                    simProps.emplace_back(
+                            "WAT_VISC",
+                            Opm::UnitSystem::measure::viscosity,
+                            adbToDoubleVector(sd.rq[aqua_idx].mu));
+                }
+                if (liquid_active) {
+                    simProps.emplace_back(
+                            "OIL_VISC",
+                            Opm::UnitSystem::measure::viscosity,
+                            adbToDoubleVector(sd.rq[liquid_idx].mu));
+                }
+                if (vapour_active) {
+                    simProps.emplace_back(
+                            "GAS_VISC",
+                            Opm::UnitSystem::measure::viscosity,
+                            adbToDoubleVector(sd.rq[vapour_idx].mu));
+                }
+            }
+
+            /**
+             * Relative permeabilities for water, oil, gas
+             */
+            if (aqua_active && outKeywords["KRW"] > 0) {
+                simProps.emplace_back(
+                        "WATKR",
+                        Opm::UnitSystem::measure::permeability,
+                        adbToDoubleVector(sd.rq[aqua_idx].kr));
+            }
+            if (aqua_active && outKeywords["KRO"] > 0) {
+                simProps.emplace_back(
+                        "OILKR",
+                        Opm::UnitSystem::measure::permeability,
+                        adbToDoubleVector(sd.rq[liquid_idx].kr));
+            }
+            if (aqua_active && outKeywords["KRG"] > 0) {
+                simProps.emplace_back(
+                        "GASKR",
+                        Opm::UnitSystem::measure::permeability,
+                        adbToDoubleVector(sd.rq[vapour_idx].kr));
+            }
+
+            /**
+             * Vaporized and dissolved gas/oil ratio
+             */
+            if (vapour_active && liquid_active && outKeywords["RVSAT"] > 0) {
+                simProps.emplace_back(
+                        "RVSAT",
+                        Opm::UnitSystem::measure::permeability,
```

Comment:
The units for RVSAT and RSSAT are also wrong, and must be fixed. (Also, I thought I saw these in the opposite order?)